frontend/src/components/form/base/__tests__/__snapshots__/EColorPicker.spec.js.snap

```diff
@@ -30,194 +30,128 @@ exports[`An EColorPicker looks like a color picker: pickerclosed 1`] = `
-  >
-     
-    <div
-      class="v-card v-sheet theme--light"
-    >
-      <div
-        class="v-color-picker v-sheet theme--light v-color-picker--flat theme--light"
-        style="max-width: 300px;"
-      >
-        <div
-          class="v-color-picker__canvas"
-          style="width: 300px; height: 150px;"
-        >
-          <canvas
-            height="150"
-            width="300"
-          />
-          <div
-            class="v-color-picker__canvas-dot"
-            style="width: 10px; height: 10px; transform: translate(295px, -5px);"
-          />
-        </div>
-        <div
-          class="v-color-picker__controls"
-        >
-          <div
-            class="v-color-picker__preview v-color-picker__preview--hide-alpha"
-          >
-            <div
-              class="v-color-picker__dot"
-            >
-              <div
-                style="background: rgb(255, 0, 0);"
-              />
-            </div>
-            <div
-              class="v-color-picker__sliders"
-            >
-              <div
-                class="v-input v-color-picker__hue v-input--hide-details theme--light v-input__slider v-color-picker__track"
-              >
-                <div
-                  class="v-input__control"
-                >
-                  <div
-                    class="v-input__slot"
-                  >
-                    <div
-                      class="v-slider v-slider--horizontal theme--light"
-                    >
-                      <input
-                        disabled="disabled"
-                        id="input-26"
-                        readonly="readonly"
-                        tabindex="-1"
-                        value="0"
-                      />
-                      <div
-                        class="v-slider__track-container"
-                      >
-                        <div
-                          class="v-slider__track-background primary lighten-3"
-                          style="right: 0px;"
-                        />
-                        <div
-                          class="v-slider__track-fill primary"
-                          style="left: 0px; width: 0%;"
-                        />
-                      </div>
-                      <div
-                        aria-orientation="horizontal"
-                        aria-readonly="false"
-                        aria-valuemax="360"
-                        aria-valuemin="0"
-                        aria-valuenow="0"
-                        class="v-slider__thumb-container grey--text text--lighten-2"
-                        role="slider"
-                        style="left: 0%;"
-                        tabindex="0"
-                      >
-                        <div
-                          class="v-slider__thumb grey lighten-2"
-                        />
-                      </div>
+<div data-app>
```

Comment:
Seltsam dass sich die Formatierung so stark unterscheidet, nur weil jetzt `<body>` nicht mehr da ist... Aber okay solange es nicht ständig hin- und her ändert.

Reply:
I suspect that most of the formatting changes arise from changing `expect(document.body).toMatchSnapshot('pickeropen')` to `expect(wrapper).toMatchSnapshot('pickeropen')`.

Reply:
This is because not the jest-serializer-vue-tjw is used if you pass document.body to match the snapshot.